src/main/java/org/folio/fqm/service/LocalizationService.java

```diff
@@ -78,7 +88,7 @@ private String getSourceTranslationPrefix(EntityType entityType, String columnNa
     }
   }
 
-  protected void localizeObjectColumn(EntityType entityType, EntityTypeColumn column, ObjectType objectColumn) {
+  private void localizeObjectColumn(EntityType entityType, EntityTypeColumn column, ObjectType objectColumn) {
```

Comment:
The visibility on these methods was bothering me, so I went ahead and locked them down as much as I could